### DISCUSSION TITLE
2011/10/19/%E4%B8%80%E4%B8%AA%E7%BF%BB%E5%A2%99%E6%B5%8F%E8%A7%88%E5%99%A8%E7%9A%84%E5%88%9B%E6%84%8F

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#348] [Announcements]
# 2011/10/19/%E4%B8%80%E4%B8%AA%E7%BF%BB%E5%A2%99%E6%B5%8F%E8%A7%88%E5%99%A8%E7%9A%84%E5%88%9B%E6%84%8F

最近，天朝有封堵了所有Wordpress的二级域名，导致我的博客无法在国内打开了。郁闷之下，我想到如果有人做一个自带翻墙功能的浏览器。

https://blog.qizhen.xyz/2011/10/19/%E4%B8%80%E4%B8%AA%E7%BF%BB%E5%A2%99%E6%B5%8F%E8%A7%88%E5%99%A8%E7%9A%84%E5%88%9B%E6%84%8F

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Colinzhang
2011年10月21日 20:14 编辑
vpn吧，不困难的，我现在就是vpn出来的。

回复
oohu
2011年10月20日 16:42 编辑
清华的紫狐，基于火狐的，好像就是了~~

回复
ruanqizhen
2011年10月20日 22:15 编辑
清华的紫狐仅仅是利用了IPV6，一是很多电脑和网站还不支持，二是网警们虽然目前没兴趣关注IPV6，但想封的话非常容易。

回复
